src/arch/xtensa/Kconfig

```diff
@@ -17,4 +17,10 @@ config SMP
 	help
 	  Indicates that architecture uses multiple cores
 
+config WAITI_WAKEUP_CALLBACKS
```

Comment:
Can you provide some examples here of usage ?

Reply:
In the description I assume, because actual use is in the next commit in the series.

Reply:
`here` == in comment on github or in kconfig? I can add note in Kconfig like `if you use this config you have to implement functions - platform_interrupt_on_wakeup & platform_interrupt_on_waiti`, but I didn't see that level of details  in our descriptions yet, so I wasn't sure if we want it.

Actually I just need `CAVS_USE_LPRO_IN_WAITI`, but it's bad to have something like `#if <CONFIG_PLATFORM_SPECIFIC_FEATURE>` in `arch` code, that's why I wanted to introduce this config to have it as arch-level feature that is used from platform code.

Reply:
@jajanusz Guess it's good if the Kconfig mentions this at least in summary, so it's clear what the feature is actually doing and why it should or shouldn't be selected in various cases.

Reply:
Yea, I guess being a bit more verbose here is good. Updated `help` with this info.